fw/application/src/i18n/language.h

```diff
@@ -12,20 +12,20 @@
 
 typedef enum {
     LANGUAGE_ZH_HANS,
-    LANGUAGE_EN_US,
     LANGUAGE_ZH_TW,
+    LANGUAGE_EN_US,
```

Comment:
please revert the order change, thanks

Reply:
Yeah, the order will impact the existing settings.bin, good catch.